src/main/resources/db/migration/V25__faktura_kreditreferanse.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE faktura
+    ADD COLUMN kredit_referanse_nr varchar(40);
```

Comment:
Var dette det beste navnet vi kunne finne? Ref. praten vi hadde på Slack.
Lurer på om du fikk en avklaring fra OEBS og vil gjerne lære mer om fagspråket her

Reply:
Vi har ikke fått svar enda. Tror ikke vi trenger å vente på dette for å merge, kan fikses med en migrering